bundles/org.openhab.binding.mqtt.espmilighthub/README.md

```diff
@@ -45,6 +45,9 @@ Leave this blank.
 **MQTT Client Status Topic:**
 `milight/status`
 
+**MQTT Client Status Messages Mode:**
+Simple.
```

Comment:
Maybe it's so simple that I don't understand it. What exactly needs to be set? Is "Simple" the option you have to choose?

Reply:
`Simple` is the option you select from the box labelled **MQTT Client Status Messages Mode:** It makes sense when you look at the GUI of the device and the rest of the setup is  formatted the same way.

Reply:
@J-N-K : is it ok for you now ?

Reply:
@J-N-K  as above are you happy?